protocol/features.json

```diff
@@ -725,6 +725,10 @@
     "milestone": "colosseo_II",
     "acs": ["0028-GOVE-188", "0028-GOVE-189", "0028-GOVE-190", "0028-GOVE-191"]
   },
+  "Volume Discount Factor Customisation": {
+    "milestone": "colosseo_II",
```

Comment:
check if this is correct

Reply:
Resolving as covered in separate fee factors PR from suzuka.